app.rb

```diff
@@ -0,0 +1,111 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rentals'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books(_books)
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+      puts ' '
+    end
+  end
+
+  def list_all_people(_people)
+    @people.each do |person|
+      print "[#{person.class}] name: #{person.name}, ID: #{person.id}, #{person.age}"
+      puts ' '
+    end
+  end
```

Comment:
- [ ] When listing people, the user is getting `Unkow` fro the student, kindly fix this

![image](https://user-images.githubusercontent.com/81932478/219879800-ffce020c-a7d8-48b1-940c-86c0f3effdfb.png)